css/styles.css

```
@@ -293,6 +293,16 @@ body {
     border-radius: 20px;
 }
 
+@media (max-width: 575.98px) {
```

```suggestion
@media (max-width: 576px) {
```

Also - as discussed - please move `@media` styling at the bottom of file and group content up together